test_lib/compaction.py

```diff
@@ -85,6 +85,15 @@ def get_compaction_strategy(node, keyspace, table):
     return compaction
 
 
+def get_table_compaction_info(keyspace: str, table: str, session: object):
```

Comment:
would be nice if this function returned dataclass with available fields (or at least ones that we need), so later could be easily used without looking into scylla docs

Reply:
looks better, in future we should expand it to support all compaction strategy classes as function name is generic.